### PR TITLE
Account number label in Swedish fix

### LINKED ISSUE #3641: Wrong bank account type in GUI
### Description

Wrong bank account type in national bank transfer (SE-SEK).  

#### Version

1.2.3

### Steps to reproduce

Konto

### Expected behaviour

NOK

### Actual behaviour

SEK

### Screenshots

Se above.

#### Additional info

"Bankgiro number" accounts are used only by business bank customers. "Kontonummer" is generic and used by private customers. Wrong name here causes misunderstandings.

### PR DESCRIPTION
**Pull request by rafaelpac**
Fixes #3641
According to this issue, the label "Bankgiro number", as it was hardcoded in BankUtil.java, should not be used in Sweden.
The label "Kontonummer" is the correct one to be used in Sweden and Norway.

Account number validation is implemented to Norwegian accounts in AccountNrValidator.java.
There is no validation implemented to Swedish accounts, so nothing to change there.
If account validation for Sweden is needed, a different method would be needed (see Wikipedia reference below):

image

### CODE REVIEW

**Comment by ghost** [comment]
Swedish account validation checks that the account number consists of:

11 or 13 or 15 decimal digits.

**Comment by ripcurlx** [comment]
@rafaelpac Could you please resolve the conflict with BankUtil and update your commit messages to follow https://chris.beams.io/posts/git-commit/#7-rules - we know that you updated BankUtil.java 😉 . Thanks!

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - https://github.com/bisq-network/bisq/pull/3645#issuecomment-556984208

**Comment by ripcurlx** [comment]
And please squash it into one commit in that case.

**Comment by camahama** [comment]
Hoping this is the correct way and place to post this bug. When I try to register a new SEPA account (Swedish Skandiabanken), the SAVE NEW ACCOUNT button remains grey and unclickable. No error messages. The same account has worked previously. Running Bisq v. 1.2.3.

**Comment by ghost** [comment]
@camahama
This is not the right place but it looks like a bug, because when you toggle SEK/EUR drop-down menu the grey out disappears.

**Comment by rafaelpac** [comment]
I'm sorry... I can't find how to change the commit message. I think it is not doable in the web interface, right?

**Comment by ripcurlx** [comment]
> I'm sorry... I can't find how to change the commit message. I think it is not doable in the web interface, right?

You have to change your commit history locally and force push your changes to your remote PR branch.
See https://www.internalpointers.com/post/squash-commits-into-one-git for reference how to do it.

File: core/src/main/java/bisq/core/locale/BankUtil.java [hunk 1]
```diff
@@ -157,9 +157,9 @@ public static String getAccountNrLabel(String countryCode) {
             case "HK":
                 return Res.get("payment.accountNr");
             case "NO":
-                return "Kontonummer"; // do not translate as it is used in Norwegian only
+                return "Kontonummer"; // do not translate as it is used in norwegian and swedish only
```
Comment (ripcurlx) [inline core/src/main/java/bisq/core/locale/BankUtil.java:160]: As it looks like the label is exactly the same I think we could merge those two cases.

**Comment by ripcurlx** [comment]
> I'm sorry... I can't find how to change the commit message. I think it is not doable in the web interface, right?

@rafaelpac Besides my comment on the cases here is how you squash multiple commits into one locally: https://www.internalpointers.com/post/squash-commits-into-one-git

**Comment by rafaelpac** [comment]
I'm sorry, I completely forgot about this PR.
This was a quick fix I did for somebody's issue about a label in Swedish, I was studying this file at that time, so it was easy for me to do it.
I don't use git locally, I only use the github web interface (only for very small fixes and to study the code, I am not a developer).
So I can't easily fix the mess I did here. It is easier for now to just close this and open a new clean one.
I will try that, I hope I don't mess it up again.

**Pull request closed without merging**